assets/styles/components/card.css

```
@@ -71,7 +71,7 @@
 }
 
 .app-card__content {
-    padding: 1.5rem;
+    padding: var(--card-content-padding-tp, 1.5rem) var(--card-content-padding-lr, 1.5rem);
```

```suggestion
    padding: var(--card-padding, 1.5rem);
```

Et dans le HTML tu peux mettre

```html
style="--card-padding: 0.5rem 1.5rem 1.5rem 1.5rem"
```

Comme ça le CSS est simple et c'est le HTML qui porte la spécificité

Sinon on va devoir créer une variable par côté...

---

Done.